wpe/src/main/java/com/wpe/wpe/PageSettings.java

```diff
@@ -0,0 +1,89 @@
+package com.wpe.wpe;
+
+import android.annotation.SuppressLint;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PageSettings
+{
+    private static final int DefaultMajorVersion = 12;
+    private static final int DefaultMinorVersion = 0;
+    private static final int DefaultBugfixVersion = 0;
+    private static String OS_VERSION;
```

Comment:
Maybe initialize it to `null` just in case.

Reply:
no need